pkg/pushsync/pushsync.go

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+// Package pushsync provides the pushsync protocol
+// implementation.
```

Comment:
that doesn't say a lot to me. Can you please be more specific on the function of pushsync?

Reply:
this is a developer guide, i don't want to document what the protocol does in this description